src/main/java/uk/gov/hmcts/reform/adoption/notification/ApplicationSubmittedNotification.java

```diff
@@ -65,7 +68,7 @@ public class ApplicationSubmittedNotification implements ApplicantNotification {
 
     @Override
     public void sendToApplicants(final CaseData caseData, final Long id) {
-        log.info("Sending application submitted notification to applicants for case : {}", id);
+        log.info("Sending APPLICANT_APPLICATION_SUBMITTED notification to applicants for case : {}", id);
```

Comment:
Use of constants names in log statement might affect readability of logs by non developers. Repeated in several places.

Reply:
For developers - yes. Otherwise I'd say its subjective personal preferences.